toolkit/scalebar/src/main/java/com/arcgismaps/toolkit/scalebar/internal/ScalebarRenderer.kt

```diff
@@ -66,10 +64,11 @@ internal fun LineScalebar(
     maxWidth: Float,
     label: String,
     colorScheme: ScalebarColors,
+    labelTypography: LabelTypography
```

Comment:
it seems that we only need the property `LabelTypography.labelStyle`, 
I think we should only pass necessary values. 
```suggestion
    labelStyle: TextStyle
```